src/main/java/org/mal_lang/langspec/ttc/TtcMultiplication.java

```diff
@@ -38,4 +43,13 @@ public double getMeanTtc() {
     double rhsMean = getRhs().getMeanTtc();
     return lhsMean * rhsMean;
   }
+
+  @Override
+  public JsonObject toJson() {
+    return Json.createObjectBuilder()
+        .add("type", "multiplication")
```

Comment:
It seems the only thing differing this class from TtcSubtraction is this argument? Could we combine and simplify?

Reply:
An alternative is to have a `static JsonObject toJson(String type)` in `TtcBinaryOperation`, and change this function to `return TtcBinaryOperation.toJson("multiplication");`. We can do the same in `StepBinaryOperation`.

Reply:
That works for me.